mason.sh

```diff
@@ -40,16 +40,18 @@ esac
 
 if [ ${MASON_PLATFORM} = 'osx' ]; then
     export MASON_HOST_ARG="--host=x86_64-apple-darwin"
-    export MASON_PLATFORM_VERSION=`xcrun --sdk macosx --show-sdk-version`
+    export MASON_PLATFORM_VERSION="all"
```

Comment:
`x86_64` instead? We're only using `all` on iOS because we're producing fat binaries for that platform; all other platforms use arch-specific platform versions.